### DISCUSSION TITLE
404 There isn't a GitHub Pages site here.

### DISCUSSION THREAD

**Discussion by gabrielfoganholi** [community/community#52354] [Accessibility]
### Select Topic Area

Question

### Body

erro 404 na hora em que coloco a url do meu site...

**Comment by neutrinomuon** [comment]
Olá, tudo bem? Eu dei uma checada na tua página (em [portfólio](https://github.com/gabrielfoganholi/portfolio)) e o site está funcionando bem. Muitas vezes quando é feito o "deploy" demora um tempo até atualizar a página. Às vezes pode ser problema do cache também. Era isso mesmo que querias? Qualquer coisa podes explicar melhor? Abraço

**Comment by ghostinhershell** [comment]
Hi @gabrielfoganholi, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
